### PR TITLE
Test more python versions in github workflow

### PR DESCRIPTION
**Pull request by bkorycki**
These changes are drawn from ModelBench.

### CODE REVIEW

**Comment by bkorycki** [comment]
Oh shoot, the hazard Enum change was meant to be in a different PR.

**Review by yifanmai — APPROVED** [review]
There's still a stray StrEnum change, otherwise LGTM

**Comment by bkorycki** [comment]
> There's still a stray StrEnum change, otherwise LGTM

@yifanmai What do you mean?

File: plugins/standard_tests/modelgauge/tests/safe.py [hunk 1]
```diff
@@ -36,7 +36,7 @@
 }
 
 
-class PersonaType(str, Enum):
+class PersonaType(StrEnum):
```
Comment (yifanmai) [inline plugins/standard_tests/modelgauge/tests/safe.py:39]: Is this change intended?

**Review by wpietri — APPROVED** [review]
If calling '.value' on the enums is all that's needed, that's probably better. But if the library's needed for more than that, I'm all for it.

**Comment by bkorycki** [comment]
The enum changes have been moved to PR #425.
